<?xml version="1.0" encoding="UTF-8"?>
<com.ibm.datatools.etl.errmsgs:Message xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:com.ibm.datatools.etl.errmsgs="http:///com/ibm/datatools/etl/errmsgs.ecore" msgNum="12208" timestamp="1517551227293" msgTxt="Validation succeeded for the 'evalp2'." msgSeverity="LOG" component="dataflow.validation" msgDescTxt="Validation succeeded for evalp2." msgCause="" msgResponse="" entityType="DataFlow">
  <expandedMsgs msgNum="12208" timestamp="1517551227293" msgTxt="Validation succeeded for the 'flowLabel'." msgSeverity="LOG" component="dataflow.validation" msgDescTxt="Validation succeeded for flowLabel." msgCause="" msgResponse="" entityType="DataFlow" operatorID="01"/>
  <expandedMsgs msgNum="12208" timestamp="1517551227293" msgTxt="Validation succeeded for the 'flowLabel'." msgSeverity="LOG" component="dataflow.validation" msgDescTxt="Validation succeeded for flowLabel." msgCause="" msgResponse="" entityType="DataFlow" operatorID="02"/>
  <expandedMsgs msgNum="12208" timestamp="1517551227293" msgTxt="Validation succeeded for the 'flowLabel'." msgSeverity="LOG" component="dataflow.validation" msgDescTxt="Validation succeeded for flowLabel." msgCause="" msgResponse="" entityType="DataFlow" operatorID="05"/>
  <expandedMsgs msgNum="12208" timestamp="1517551229356" msgTxt="Validation succeeded for the 'flowLabel'." msgSeverity="LOG" component="dataflow.validation" msgDescTxt="Validation succeeded for flowLabel." msgCause="" msgResponse="" entityType="DataFlow">
    <expandedMsgs msgNum="54003" timestamp="1517551229356" msgTxt="Error happens when processing operator: &quot;Group By_02&quot;, code generation could be non-optimal or problematic: When processing statement:&quot; SELECT &quot;INPUT_02_0&quot;.&quot;CUSTID&quot;  AS &quot;CUSTID&quot;, &quot;INPUT_02_0&quot;.&quot;COUNTRY&quot;  AS &quot;COUNTRY&quot;  FROM  &quot;INPUT_02_0&quot;  GROUP BY &quot;INPUT_02_0&quot;.&quot;COUNTRY&quot; HAVING count (&quot;INPUT_02_0&quot;.&quot;CUSTID&quot; ) > 2&quot; Projection column CUSTID not in group by list " msgSeverity="WARN" component="dataflow.codegen" msgDescTxt="" msgCause="Error happens when generating internal code generation structure." msgResponse="Make sure that all properties are properly set." entityType="DATAFLOW"/>
  </expandedMsgs>
</com.ibm.datatools.etl.errmsgs:Message>
